WordPressAuthenticatorTests/Authenticator/PasteboardTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  PasteboardTests.swift
+//  WordPressAuthenticatorTests
+//
+//  Created by Allen Snook on 1/13/21.
+//  Copyright © 2021 Automattic. All rights reserved.
+//
```

Comment:
Can you remove this generated header?

We generally keep them out of the projects, but there's no good linter rule available to do it. 😞

Reply:
Fixed in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/561/commits/c96f997731e9859ef5b96e0a8e5561a4683869ae